pyfr/writers/paraview.py

```diff
@@ -63,6 +65,75 @@ def write_out(self):
         write_s('\n</AppendedData>\n</VTKFile>')
 
 
+class ParallelParaviewWriter(BaseWriter):
+    """Wrapper for writing serial .vtu Paraview files"""
+    # Supported file types and extensions
+    name = 'paraview'
+    extn = ['.pvtu']
+
+    # PyFR to VTK element types and number of nodes
+    vtk_types = {
+        'tri': (5, 3), 'quad': (9, 4),
+        'tet': (10, 4), 'pyr': (14, 5), 'pri': (13, 6), 'hex': (12, 8)
+    }
+
+    def write_out(self):
+        """Controls the writing of serial .vtu Paraview files
+
+        Writes .vtu pieces for each element type, in each partition of
+        the PyFR files.  The Paraview data type used is "appended",
+        which concatenates all data into a single block of binary data
+        at the end of the file.  ASCII headers written at the top of
+        the file describe the structure of this data.
+        """
+        # Set default divisor to solution order
+        if self.args.divisor == 0:
+            self.args.divisor = self.cfg.getint('solver', 'order')
+
+        write_s = lambda s: self.outf.write(s.encode('utf-8'))
+
+        basename = os.path.splitext(self.outf.name)[0]
+
+        write_s('<?xml version="1.0" ?>\n<VTKFile '
+                'byte_order="LittleEndian" type="PUnstructuredGrid" '
+                'version="0.1">\n<PUnstructuredGrid>\n')
+
+        for pi, (mk, sk) in enumerate(zip(self.mesh_inf, self.soln_inf)):
+            pfn = basename+'_part{}.vtu'.format(pi)
```

Comment:
Use of '+' for string building is usually not appropriate (PEP8).  Use `'{0}_part{1}'.format(basename, pi)`.